app/containers/SongDetailContainer/saga.js

```diff
@@ -0,0 +1,20 @@
+import { put, call, takeLatest } from 'redux-saga/effects';
+import { songDetailContainerTypes, songDetailContainerCreators } from './reducer';
+import { getSingleItune } from '@app/services/ituneApi';
+
+// Individual exports for testing
+const { REQUEST_GET_SINGLE_ITUNE } = songDetailContainerTypes;
+const { successGetSingleItune } = songDetailContainerCreators;
+
+export function* getItune(action) {
+  const response = yield call(getSingleItune, action.trackId);
+  const { data, ok } = response;
+  /* istanbul ignore else */
```

Comment:
why did we add this?

Reply:
Since we do not have any else branch, we are telling jest to just ignore the else branch.